Update settings.md (#299)
nuejs/nue: packages/nuejs.org/docs/settings.md

### port
The port number of the development server. Default is 8080.

### syntax_highlight
Set this to `false` to disable the [built-in stylesheet](syntax-highlighting.html) for code blocks.

### view_transitions
Setting this to `true` enables [view transitions](reactivity.html#view-transitions) for instant and smooth page switches.

## Site settings
List of site-wide settings that impact your SEO data and other metadata inside your document `<head>` element. These are defined in your `site.yaml` but can be overwritten on area and page level.

### base
The value of HTML [<base>](//developer.mozilla.org/en-US/docs/Web/HTML/Element/base) attribute.

### charset
The value of the charset meta tag. The default is "utf-8".

### direction
The value of `<html direction="{ direction }">` attribute. The default is "ltr" (left to right).

### favicon
Relative path to your favicon that overrides the "favicon.ico" on the browser tab.

### generator

> The generator meta tag. The default is "Nue (nuejs.org)" and is only rendered on the production pages.

Reviewer: ```suggestion
The generator meta tag. The default is "Nue (nuejs.org)" and is only rendered in production mode.
```
(I think `for` fits best, but I understand, that `on` emphasizes, that it's in the page's html)

Or maybe just `in production (mode)`?

Author: I think in production (mode) seems the best choice.

Reviewer: Okay. 
Do you like `in production mode.` or `in production.` better?

Author: I think in production mode makes it clearer what it is.
